README.md

```diff
@@ -34,6 +34,7 @@ python C:\Users\pySearch\pySearch.py  ( different path name where you have clone
  <li> -e  changes name or alias of search engine and sets it to as search engine for session </li> 
  <li> -d  changes to domain extension </li>
  <li> -h  will provide description of command line arguments </li> 
+ <li> -b changes the browser to search in. If an invalid browser is selected, a list of valid ones will be displayed.</li>
```

Comment:
Could we mention that on Windows, browsers are only registered IF the path for the browser's install directory is added to the system's environment variables?